ephios/core/signals.py

```diff
@@ -30,6 +30,13 @@
 Receivers will receive a ``request`` keyword argument.
 """
 
+nav_link = PluginSignal()
+"""
+This signal is sent out to get links for the main navbar. Receivers should return a dict of
+with keys being the text and values being the url to link to.
+Receivers will receive a ``request`` keyword argument.
```

Comment:
I think this should work similar to the administration links, where we include the info of whether the link is active. This allows us to later visually show the "active" nav item, should we ever want to.